pages/vi/vi-planet-installation-and-configuration.md

```diff
@@ -134,7 +134,13 @@ Follow the official [Install Docker Compose](https://docs.docker.com/compose/ins
 
 ## Planet Configuration
 
-Go to your community `planet` at <http://127.0.0.1>. You will be asked to:
+Go to your community `planet` at <http://127.0.0.1>.
+
+ If you are using docker toolbox then you should use the IP given in the docker tool box
```

Comment:
Please change this line to
>For **Windows**, if you are using docker toolbox then you should use the *IP Address* given in the docker toolbox instead of `127.0.0.1`. Please look for *IP Address* as seen in image below.